### PR TITLE
Add api doc

### PR DESCRIPTION
**Pull request by MikhailAlferov**

### CODE REVIEW

File: API.en.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+This class is designed to actively receive information from the Telegram server and has the following methods:  
+  
+`login (AuthKey $ authKey [, Proxy $ proxy [, callable $ cb]])` - authorization of the client on the Telegram server.  
+* *$authKey* - authorization key received after registering the phone number on the Telegram server.  
+* *$proxy* - an object for using a proxy server to operate the library  
+* *$cb* - a callback function that will be executed after authorization.  
+  
+`isLoggedIn ()` - client authorization check  
+  
+`pollMessage ()` - a method awaiting receipt of messages from the Telegram server
```
Comment (tombouctou) [inline API.en.md]: remove `a method`

File: API.en.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+`getChatMembers (int $ id, callable $ onComplete)` - getting chat participants.  
+* *$id* - chat id  
+* *$onComplete* - a callback function, in the input parameter of which comes AnonymousMessage - a message from the server with the requested information  
+  
+`getChannelMembers (int $ id, int $ accessHash, callable $ onComplete)` - getting channel members  
+* *$id* - channel id  
+* *$accessHash* - hash coming from the server in the ‘access_hash’ channel information field  
+* *$onComplete* - a callback function, in the input parameter of which comes AnonymousMessage - a message from the server with the requested information  
+  
+`getFullChannel (int $ id, int $ accessHash, callable $ onComplete)` - ???
```
Comment (tombouctou) [inline API.en.md]: why spaces before `$`?
the description can be:
```
get channel info with all settings (notifications, permissions, photo, etc.)

File: API.en.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+`pollMessage ()` - a method awaiting receipt of messages from the Telegram server  
+  
+`getChatMembers (int $ id, callable $ onComplete)` - getting chat participants.  
+* *$id* - chat id  
+* *$onComplete* - a callback function, in the input parameter of which comes AnonymousMessage - a message from the server with the requested information  
+  
+`getChannelMembers (int $ id, int $ accessHash, callable $ onComplete)` - getting channel members  
+* *$id* - channel id  
+* *$accessHash* - hash coming from the server in the ‘access_hash’ channel information field  
+* *$onComplete* - a callback function, in the input parameter of which comes AnonymousMessage - a message from the server with the requested information
```
Comment (tombouctou) [inline API.en.md]: with requested information

File: API.en.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
+
+
+## InfoClient
+
+This class is designed to actively receive information from the Telegram server and has the following methods:  
+  
+`login (AuthKey $ authKey [, Proxy $ proxy [, callable $ cb]])` - authorization of the client on the Telegram server.  
+* *$authKey* - authorization key received after registering the phone number on the Telegram server.  
+* *$proxy* - an object for using a proxy server to operate the library
```
Comment (wkejqb) [inline API.en.md]: `proxy server abstraction` would be simpler

File: API.en.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
+
+
+## InfoClient
+
+This class is designed to actively receive information from the Telegram server and has the following methods:  
+  
+`login (AuthKey $ authKey [, Proxy $ proxy [, callable $ cb]])` - authorization of the client on the Telegram server.  
+* *$authKey* - authorization key received after registering the phone number on the Telegram server.  
+* *$proxy* - an object for using a proxy server to operate the library  
+* *$cb* - a callback function that will be executed after authorization.
```
Comment (wkejqb) [inline API.en.md]: callback function to be executed after authorization

File: API.en.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+* *$onComplete* - a callback function, in the input parameter of which comes AnonymousMessage - a message from the server with requested information  
+  
+`getChannelMembers (int $id, int $accessHash, callable $onComplete)` - getting channel members  
+* *$id* - channel id  
+* *$accessHash* - hash coming from the server in the ‘access_hash’ channel information field  
+* *$onComplete* - a callback function, in the input parameter of which comes AnonymousMessage - a message from the server with requested information  
+  
+`getFullChannel (int $id, int $accessHash, callable $onComplete)` - get channel info with all settings (notifications, permissions, photo, etc.)  
+  
+`getChatMessages (int $id, int $limit, int $since, int $lastId, callable $onComplete)` - receive messages from the chat with the ability to filter by time and quantity.
```
Comment (wkejqb) [inline API.en.md]: `from chat, filtered by time sent; $limit, max number of messages expected to be received, is <= 100`

**Pull request merged**